Documentation/ProjectFile/prj/processes/process/TWOPHASE_FLOW_PP/material_property/fluid/i_fluid.md

```diff
@@ -0,0 +1 @@
+A tag for fluid properties of gas and liquid phases.
```

Comment:
Style: `A tag for` doesn't sound so well. Maybe instead use something like `Defines fluid properties of gas and liquid phases.`